shared/xccdf/services/imap.xml

```diff
@@ -23,7 +23,7 @@ POP3 server, the dovecot software should be disabled and removed.
 Running an IMAP or POP3 server provides a network-based
 avenue of attack, and should be disabled if not needed.
 </rationale>
-<ident prodtype="rhel7" cce="80294-2" />
+<ident prodtype="rhel7" cce="80294-2" cis="2.2.11" />
```

Comment:
The log complains about ident without required attribute 'system', which is weird. The only change to `ident` is here.
```
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/ssg_new/build/RHEL/7/xccdf-unlinked-resolved.xml' line 31291: Element '{http://checklists.nist.gov/xccdf/1.1}ident': The attribute 'system' is required but missing.
OpenSCAP Error: Invalid XCCDF Checklist (1.1) content in /home/jenkins/workspace/scap-security-guide-pull-requests/label/ssg_new/build/RHEL/7/xccdf-unlinked-resolved.xml. [oscap_source.c:342]
```

Shouldn't`cis="2.2.11` go to `ref` element?

Reply:
You are correct. Thanks for pointing that out. I'm still learning what all the error messages mean and how to find it.